WhoIsThat/WhoIsThat/Connections/RestService.cs

```diff
@@ -46,5 +46,23 @@ public async Task<List<ImageObject>> GetImageObjects()
                 throw exception;
             }
         }
+
+        public async Task<string> Identify()
+        {
+            string restUrl = "https://testrecognition.azurewebsites.net/api/recognitionservices/identify";
+            var uri = new Uri(string.Format(restUrl, string.Empty));
```

Comment:
I know it's too late, but saw it in my email:D why are you using format to add nothing?